receiver/mysqlreceiver/metadata.yaml

```diff
@@ -283,6 +287,16 @@ metrics:
       monotonic: false
       aggregation: cumulative
     attributes: [threads]
+  mysql.bytes.count:
```

Comment:
Typically we should not use the unit in the name. Instead, I think `throughput` is usually appropriate.

Reply:
Changed to `mysql.client_throughput.count`

Reply:
I believe I erred with my suggestion of using `throughput`. This type of metric is quite specifically called out in the naming guidelines as a [`.io` metric](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/README.md#instrument-naming)?